plugins/org.yakindu.sct.doc.user/src/user-guide/c-domain.textile

```diff
@@ -770,6 +770,109 @@ static void statechart_enact_main_region_Check(Statechart* handle)
 }
 
 
+h2(#cdom_annotations_short_identifiers). The @ShortIdentifiers annotation
+
+It is possible to add annotations to a statechart's specification to alter their or the code generator's behavior, see "Statechart Annotations":../user-guide/statechart_language.textile#sclang_annotations.
+
+The C Domain offers one more annotation: @ShortIdentifiers helps you to keep the generated code compliant to rules which require identifiers to be shorter than 31 characters.
+
+This is mainly done by:
```

Comment:
Explain the strategy: The user will be in charge and control of choosing appropriate identifiers in the statechart in order to get appropriate identifiers in the C code. This includes direct feedback using C identifier checking ...